armory/data/datasets.py

```diff
@@ -730,11 +731,34 @@ def imagenette(
     )
 
 
+def gtsrb_canonical_preprocessing(img):
+    img_size = 48
+    img_out = []
+    quantization = 255.0
+    for im in img:
+        img_eq = ImageOps.equalize(Image.fromarray(im))
```

Comment:
I'm not sure about doing equalization inside of the canonical preprocessing step - that can significantly change an image and is not a simple thing to reverse.